tests/phpunit/integration/tests/Media/SVG.php

```diff
@@ -23,44 +23,58 @@
  * @coversDefaultClass \Google\Web_Stories\Media\SVG
  */
 class SVG extends TestCase {
+	/**
+	 * Test instance.
+	 * @var \Google\Web_Stories\Media\SVG
+	 */
+	protected $instance;
+
+	public function set_up() {
+		parent::set_up();
+
+		$experiments = $this->createMock( \Google\Web_Stories\Experiments::class );
+		$experiments->method( 'is_experiment_enabled' )
+					->willReturn( true );
+
+		$this->instance = new \Google\Web_Stories\Media\SVG( $experiments );
```

Comment:
I don't like this. It is much cleaner to create a new instance of a object every time for each test. So there no problem of state being from previous test. The test was better the way it was.

Reply:
I was doing some cleanup like this while working on this PR to make things more DRY.

Note that this is in `set_up` so we still get a new instance for every test. No need to repeat this when `set_up` can do this once for every test.

So I disagree with the "The test was better the way it was"